### PR TITLE
Fix API bugs

### PR DESCRIPTION
**Pull request by MGlauer**
* Fix typing issues with different date/time formats
* Implement type specification in value clauses
* Fix rounding issues with `float`

### CODE REVIEW

File: api/actions.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def get_table_name(schema, table, restrict_schemas=True):
     if schema.startswith("_") or schema == "public" or schema is None:
         raise PermissionDenied
     if restrict_schemas:
-        if schema not in ["model_draft", "sandbox", "test"]:
+        if schema not in PLAYGROUNDS and schema not in UNVERSIONED_SCHEMAS:
```
Comment (Bachibouzouk) [inline api/actions.py]:
```suggestion
        if schema not in PLAYGROUNDS + UNVERSIONED_SCHEMAS:
```

File: api/actions.py [hunk 2]
```diff
@@ -1090,7 +1020,7 @@ def data_delete(request, context=None):
     setter = []
     cursor = load_cursor_from_context(context)
     result = __change_rows(request, context, target_table, setter, ["id"])
-    if orig_schema in PLAYGROUNDS:
+    if orig_schema in PLAYGROUNDS or orig_schema in UNVERSIONED_SCHEMAS:
```
Comment (Bachibouzouk) [inline api/actions.py]:
```suggestion
    if orig_schema in PLAYGROUNDS + UNVERSIONED_SCHEMAS:
```

File: api/actions.py [hunk 3]
```diff
@@ -1116,7 +1046,7 @@ def data_update(request, context=None):
         setter = dict(zip(field_names, setter))
     cursor = load_cursor_from_context(context)
     result = __change_rows(request, context, target_table, setter)
-    if orig_schema in PLAYGROUNDS:
+    if orig_schema in PLAYGROUNDS or orig_schema in UNVERSIONED_SCHEMAS:
```
Comment (Bachibouzouk) [inline api/actions.py]:
```suggestion
    if orig_schema in PLAYGROUNDS + UNVERSIONED_SCHEMAS:
```

File: api/actions.py (hunk 1 above)
Comment (MGlauer) [reply api/actions.py]: I am not sure which version I prefer. The first version performs better for large lists. Yours is undoubtedly more readable  and - according to `timeit` - more efficient for small lists. Interesting. As I expect small sets of schemas, I will go with your version.

**Pull request merged**